code/modules/atmospherics/machinery/other/miner.dm

```diff
@@ -175,3 +177,60 @@
 	name = "\improper Water Vapor Gas Miner"
 	overlay_color = "#99928E"
 	spawn_id = /datum/gas/water_vapor
+
+/obj/machinery/atmospherics/miner/station
+	power_draw = GASMINER_POWER_FULLSCALE
+	spawn_mol = MOLES_CELLSTANDARD / 10
+	desc = "This item should not exist, file a bug report if you see this message"
```

Comment:
???

Reply:
Is there a better solution to only have the text description in one place?
The other ones are mapping items, and the `/obj/machinery/atmospherics/miner/station` should never be mapped in, only the subtypes of it.

Reply:
well it won't work anyway, it doesnt need the desc like this

Reply:
Visibly broken instead of silently